### ISSUE TITLE
$config['message_stream_id'] not defined?

### ISSUE THREAD

**Issue by bhu-gmg** [coconutcraig/laravel-postmark#141]
`$config['message_stream_id']` is not defined, and so it triggers a PHP error - maybe it should be `$config['message_stream_id'] ?? ''`, or something similar. Am I meant to be defining `message_stream_id` somewhere? It feels like a default (i.e. not defining it) should be ok.

- Laravel Postmark Version: 3.0.4
- Laravel Version: 10.15.0
- PHP Version: 8.1.19

**Comment by fvanzeijl** [comment]
Default config from laravel for postmark-mail does not include message_stream_id. 
The changes made in last 2 commits are provide unchecked config-parameter, so it will break.
The token (one line after) called with `$this->app->make('config')->get('services.postmark.token')` so why not `message_stream_id` is called the same way?

Until the pull-request is pending or will not be approved you can add the key to your mail-config.

config/mail.php
```php
// ...
        'postmark' => [
            'transport' => 'postmark',
            'message_stream_id' => null
        ],
```

**Comment by craigpaul** [comment]
Sorry about that, went too fast with that last change. This has been resolved in 0f20810. As for why it's not called the same way (with the configuration repository) please see #140

**Comment by bhu-gmg** [comment]
@craigpaul thx for the quick fix
